Add utm 2

sumitshinde-84 opened the pull request:
## Description

<!-- Describe your changes in detail -->

## Related Issue(s)

<!-- What issue does this PR relate to? -->

## Checklist

<!-- https://flowfuse.com/handbook/development/#defining-done -->

 - [ ] I have read the [contribution guidelines](https://github.com/FlowFuse/flowfuse/blob/main/CONTRIBUTING.md)
 - [ ] I have considered the performance impact of these changes
 - [ ] Suitable unit/system level tests have been added and they pass
 - [ ] Documentation has been updated

ZJvandeWeg commented on src/blog/2024/04/building-an-admin-panel-in-node-red-with-dashboard-2.md:
```diff
@@ -98,3 +98,4 @@ If you want to learn more about FlowFuse multiuser addon and personalize the mul
 
 - [Multi-User Dashboard for Ticket/Task Management](/blueprints/flowfuse-dashboard/multi-user-dashboard/) blueprint, which allows you to utilize templates to develop Personalize multi-user dashboard quickly.
 
+If you are looking to build a multi-user dashboard, deploy it in seconds, scale and manage Node-RED efficiently, and enable seamless remote access for your entire team, [sign up](https://app.flowfuse.com/account/create?utm_campaign=60718323-BCTA&utm_source=blog&utm_medium=cta&utm_term=high_intent&utm_content=How%20to%20Build%20an%20Admin%20Dashboard%20with%20Node-RED%20Dashboard%202.0) for FlowFu ...
```
Why isn't this a partial?

sumitshinde-84 replied:
like this:  `{{ site.appURL }}/account/create......`

ZJvandeWeg replied:
No, more like: https://cloudcannon.com/tutorials/eleventy-beginner-tutorial/eleventy-partials/

ZJvandeWeg replied:
So if the CTA is standard across many articles, let's include a partial to faster iterate on the CTA

sumitshinde-84 commented:
@Yndira-E could you please add the correct styling for the cta link

Yndira-E commented:
@sumitshinde-84 The issue isn't with the classes; it's that `<p>` tags are being created inside the `<a>` tags, which causes the styles to appear as paragraphs.

<img width="611" alt="Screenshot 2025-03-04 at 15 20 21" src="https://github.com/user-attachments/assets/a2817b63-1298-4cff-960f-3375cd533261" />

I added a class to address that specific structure, but it's not the best approach. The issue seems to be related to the if statements. If I remove them, the `<p>` tags aren't created.

<img width="591" alt="Screenshot 2025-03-04 at 15 44 33" src="https://github.com/user-attachments/assets/726bb180-fe51-41b0-ba0a-aa22e98c4f20" />

Maybe it would be better to create three partials?

sumitshinde-84 commented:
> Maybe it would be better to create three partials?

I have fixed the structure, and now default style is working correctly.

Yndira-E commented:
@sumitshinde-84 I checked your version and it's creating four `<a>` tags, and three of them have a URL but no text.

<img width="601" alt="Screenshot 2025-03-04 at 16 20 02" src="https://github.com/user-attachments/assets/606c393c-c398-4f88-9889-efc5bfe5b3f6" />


I've tested the following structure and it works without the need of that extra class I created:

```
<div class="cta">
  {% if cta_text and cta_text != "" %}
    {{ cta_text }}
  {% else %}
    Optimize your factory operations with FlowFuse. Effortlessly manage, deploy, scale, and secure your Node-RED instances.
  {% endif %}

  {% assign link_text = "" %}
  {% if cta_type == "talk" %}
    {% assign link_text = "Talk to Us Now" %}
  {% elsif cta_type == "demo" %}
    {% assign link_text = "Let’s Walk You Through It – Book a Demo Today" %}
  {% else %}
    {% assign link_text = "Sign Up for FlowFuse Now" %}
  {% endif %}

  <a href="{{ cta_url }}">{{ link_text }}</a>
</div>
```
<img width="601" alt="Screenshot 2025-03-04 at 16 20 02" src="https://github.com/user-attachments/assets/c2037754-e126-40b7-8c00-35d87c009a69" />


You need to move the if statements outside the <a> tags and use the assign statement to set the link text based on the conditions. This ensures that the link text is correctly assigned without wrapping it in a paragraph, maintaining valid HTML structure.

Yndira-E commented:
I've just removed the unnecessary tags from the partial, it was creating empty `p` tags. 
<img width="720" alt="Screenshot 2025-03-04 at 17 14 57" src="https://github.com/user-attachments/assets/012ad9b1-34ed-4733-b6d3-269108182055" />

Yndira-E commented on src/blog/2024/04/building-an-admin-panel-in-node-red-with-dashboard-2.md:
```diff
@@ -98,3 +98,4 @@ If you want to learn more about FlowFuse multiuser addon and personalize the mul
 
 - [Multi-User Dashboard for Ticket/Task Management](/blueprints/flowfuse-dashboard/multi-user-dashboard/) blueprint, which allows you to utilize templates to develop Personalize multi-user dashboard quickly.
 
+{% include "cta.njk", cta_url: "https://app.flowfuse.com/account/create?utm_campaign=60718323-BCTA&utm_source=blog&utm_medium=cta&utm_term=high_intent&utm_content=How%20to%20Build%20an%20Admin%20Dashboard%20with%20Node-RED%20Dashboard%202.0", cta_type: "signup", cta_text: "Looking to build a multi-user dashboard, deploy it in seconds, scale and manage Node-RED efficiently, and enable seamless rem ...
```
@sumitshinde-84  The indentation of the text block here is off, I think it should be the same as the link. I didn't check every file, but I'd recommend you to take a look to make sure they all look ok.
<img width="745" alt="Screenshot 2025-03-04 at 17 23 38" src="https://github.com/user-attachments/assets/643b3cd2-fd70-402b-a741-672ec9d5a037" />

sumitshinde-84 commented on src/blog/2024/07/building-on-flowfuse-devices.md:
```diff
@@ -92,7 +92,7 @@ Given the above architecture, let's take a look at the relevant FlowFuse offerin
 ### Deploy to FlowFuse
 
-If you're interested in discussing how your company could benefit from this design pattern, please do [get in touch](/contact-us/).
+If you're interested in discussing how your company could benefit from this design pattern, [talk to us](https://flowfuse.com/contact-us/?utm_campaign=60718323-BCTA&utm_source=blog&utm_medium=cta%20contact%20us&utm_term=high_intent&utm_content=Building%20on%20FlowFuse%3A%20Remote%20Device%20Monitoring).
```
No, I have kept it intentionally because cta.njk is not rendering in this file,  not sure why, maybe because of the Eleventy used at the end.

sumitshinde-84 commented:
> I've just removed the unnecessary tags from the partial, it was creating empty `p` tags. <img alt="Screenshot 2025-03-04 at 17 14 57" width="720" src="https://private-user-images.githubusercontent.com/129537638/419101360-012ad9b1-34ed-4733-b6d3-269108182055.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDExMTI2MzIsIm5iZiI6MTc0MTExMjMzMiwicGF0aCI6Ii8xMjk1Mzc2MzgvNDE5MTAxMzYwLTAxMmFkOWIxLTM0ZWQtNDczMy1iNmQzLTI2OTEwODE4MjA1NS5wbmc_WC1BbXotQWxnb3JpdGhtPUFXUzQtSE1BQy1TSEEyNTYmWC1BbXotQ3JlZGVudGlhbD1BS0lBVkNPRFlMU0E1M1BRSzRaQSUyRjIwMjUwMzA0JTJGdXMtZWFzdC0xJTJGczMlMkZhd3M0X3JlcXVlc3QmWC1BbXotRGF0ZT0yMDI1MDMwNFQxODE4NTJaJlgtQW16LUV4cGlyZXM9MzAwJlgtQW16LVNpZ25hdHVyZT04ZmI3Y2RhZjhiOTgyMWUyYmY5ZTBmYTM0ODA4OWViYjQzZDQyY2U2MzQ0NGIwZjhhNGNmMzFjOThiYjliYzdmJlgtQW16LVNpZ25lZEhlYWRlcnM9aG9zdCJ9.wzki-3FbrRH1NN_mIyUFBEUjkU_3J2Lkwb7eiKwSTnU">

It needs to be inside the div container otherwise, if there is a list at the end of the blog, the CTA text will go inside the list's ul tag, causing [this](https://github.com/FlowFuse/website/pull/2996#discussion_r1979802780) indentation issue. I have added the div back.

ZJvandeWeg commented:
@Yndira-E Approved frm my side, you can merge if you feel it's ready.
